### ISSUE TITLE
Common problems of NPK compound fertilizer production

### ISSUE THREAD

**Issue by FertilizerGranulator** [FertilizerGranulator/Fertilizer-Equipment#31]
<p class="MsoNormal">
	NPK compound fertilizer is synthesized from chemical raw materials needed by various plants. NPK fertilizer has played a significant role in increasing production and income of agricultural production. So what are the common problems in the <a href="https://www.fertilizergranulator.com/production-line/npk-fertilizer-production-line.html"><u>NPK fertilizer production process</u></a>&nbsp;and what needs to be known about the equipment for the production of NPK fertilizer?
</p>
<p class="MsoNormal">
	&nbsp;
</p>
<p class="MsoNormal">
	Specialized NPK fertilizer production equipment is naturally used for fertilizer production, but due to the nature of compound fertilizer itself, fertilizer equipment is easy to be corroded, and some frequently used equipment has been seriously corroded in less than a year. Normal loss is inevitable, but necessary equipment maintenance can prolong the service life of fertilizer processing equipment.
</p>
<p class="MsoNormal">
	&nbsp;
</p>
<p class="MsoNormal">
	When crushing raw materials, we should try our best to ensure that there are no large hard blocks in the raw materials. In the compound fertilizer raw materials, if there are stones or even iron blocks, it will cause serious damage to the high-speed rotating mill rotor. Only by ensuring that the raw materials are clean and free of impurities can the equipment of fertilizer production process not be affected.
</p>
<p class="MsoNormal">
	&nbsp;
</p>
<p class="MsoNormal">
	Mechanical equipment relies on motor and reducer to obtain power source. Periodically adding lubricating oil can better ensure their operation. There are many types of lubricating oil, so we should choose suitable grease according to the principle of climate and mechanical transmission. For example, in cold areas and hot and humid environment, it is necessary to select the appropriate lubricant to ensure better operation of the equipment. In addition, when filling oil, control the amount of oil, so as not to affect the effect too little, or too much waste.
</p>
<p class="MsoNormal">
	&nbsp;
</p>
<p class="MsoNormal">
	Different equipments have different productivity. We should use the equipments reasonably according to their productivity. For example, in the compounding stage of <a href="https://www.fertilizergranulator.com/production-line/"><u>fertilizer manufacturing process</u></a>, forklift trucks should be allowed to drop materials slowly when feeding into silos, so as not to damage the equipment. When mixing, the raw materials should not be too full, and materials should be added after the start of the mixer, in order to avoid equipment damage due to excessive stirring resistance.&nbsp;
</p>
<p class="MsoNormal">
	<img src="https://www.fertilizergranulator.com/d/pic/info/npk-fertilizer-production-process.jpg" title="NPK Fertilizer Production Process" alt="NPK Fertilizer Production Process" />
</p>
<p class="MsoNormal">
	&nbsp;
</p>
<p class="MsoNormal">
	In short, there are many details about the production of NPK fertilizer. Only by maintaining the fertilizer production equipment can they work better and bring us more benefits.
</p>
